about.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-0GGDWZYGDE"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        window.dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "XXXXX");
+    </script>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Capstone</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body class="about-body">
+    <div class="optionsheadercontainer">
+      <ul class="optionsheader">
+        <li><i><img class="icon1" src="./imgs/facebook-icon-png-774.png" alt="" href="#"></i></li>
+        <li><i><img class="icon2" src="./imgs/Twitter.png" alt="" href="#"></i></li>
+        <li class="optiontext1"><a href="#">English</a></li>
+        <li class="optiontext2"><a href="#">log in</a></li>
+      </ul>
+    </div>
+      <header class="header">
+          <div class="headerelement headerelement1"></div>
+          <nav class="headerelement headerelement2">
+            <div class="lines">
+              <div class="menuline line-1"></div>
+              <div class="menuline line-2"></div>
+              <div class="menuline line-3"></div>
+            </div>
+            <ul class="menuitems">
+              <li class="menuitem menuitem-1">
+                  <a href="./index.html">Home</a>
+              </li>
+              <li class="menuitem menuitem-2">
+                <a href="./about.html">About</a>
+              </li>
+              <li class="menuitem menuitem-3">
+                <a href="#">lorem</a>
+              </li>
+              <li class="menuitem menuitem-4">
+                <a href="#">Ipsum</a>
+              </li>
+            </ul>
+            <div class="navmenubottom"></div>
+          </nav>
+          <nav class="header-desktop">
+            <div class="headerimgcontainer"><a href="./index.html"><img class="headerimg" src="./imgs/E3logowhite.png" alt=""></a></div>
+            <ul class="headercontainer">
+              <li class="header-desktop1"><a href="./index.html">Home</a></li>
+              <li class="header-desktop2"><a href="./about.html">About</a></li>
+              <li class="header-desktop3"><a href="#">Lorem</a></li>
+              <li class="header-desktop4"><a href="#">Ipsum</a></li>
+            </ul>
+            <div class="hbuttoncontainer"><button class="headerbutton">News</button></div>
+          </nav>
```

Comment:
- [ ] The navbar on desktop size have the elements centered, it would be great if these elements are aligned to the right side according to the given design, please fix this. 🌵 
![image](https://user-images.githubusercontent.com/20563252/128289578-7635d0b8-5c5d-4044-b507-f1de7e156ca3.png)

Reply:
Ok 😄  It's done!